### PR TITLE
Bump version to 0.16 and update changelog

### PR DESCRIPTION
**Pull request by tamirms**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 As this project is pre 1.0, breaking changes may happen for minor version bumps. A breaking change will get clearly notified in this log.
 
+## 0.16.0
+
+* Update XDR definitions and auto-generated classes to support upcoming protocol 13 release. (https://github.com/stellar/java-stellar-sdk/pull/276)
+* Extend StrKey implementation to handle [CAP 27 Muxed Accounts](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0027.md). (https://github.com/stellar/java-stellar-sdk/pull/276)
```
Comment (abuiles) [inline CHANGELOG.md:8]:
```suggestion
* Extend StrKey implementation to handle [CAP 27 Muxed Accounts](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0027.md) ([#276](https://github.com/stellar/java-stellar-sdk/pull/276)).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+* Update `TransactionResponse` to include new fields which are relevant to [CAP 15 Fee-Bump Transactions](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0015.md). (https://github.com/stellar/java-stellar-sdk/pull/275)
```
Comment (abuiles) [inline CHANGELOG.md:9]:
```suggestion
* Update `TransactionResponse` to include new fields which are relevant to [CAP 15 Fee-Bump Transactions](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0015.md) ([#275](https://github.com/stellar/java-stellar-sdk/pull/275)).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,18 @@
...
+* Update `AccountResponse.Balance`, `AllowTrustOperationResponse`, and create `TrustlineAuthorizedToMaintainLiabilitiesEffectResponse` to support [CAP 18 Fine-Grained Control of Authorization](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0018.md). (https://github.com/stellar/java-stellar-sdk/pull/274)
```
Comment (abuiles) [inline CHANGELOG.md:10]:
```suggestion
* Update `AccountResponse.Balance`, `AllowTrustOperationResponse`, and create `TrustlineAuthorizedToMaintainLiabilitiesEffectResponse` to support [CAP 18 Fine-Grained Control of Authorization](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0018.md) ([#274](https://github.com/stellar/java-stellar-sdk/pull/274)).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,18 @@
...
+* Add `FeeBumpTransaction` and `FeeBumpTransaction.Builder` for parsing and creating [CAP 15 Fee-Bump Transactions](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0015.md). (https://github.com/stellar/java-stellar-sdk/pull/278)
```
Comment (abuiles) [inline CHANGELOG.md:11]:
```suggestion
* Add `FeeBumpTransaction` and `FeeBumpTransaction.Builder` for parsing and creating [CAP 15 Fee-Bump Transactions](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0015.md) ([#278](https://github.com/stellar/java-stellar-sdk/pull/278)).
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,18 @@
...
+* Add methods to `Server` for submitting [CAP 15 Fee-Bump Transactions](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0015.md). (https://github.com/stellar/java-stellar-sdk/pull/278)
```
Comment (abuiles) [inline CHANGELOG.md:12]:
```suggestion
* Add methods to `Server` for submitting [CAP 15 Fee-Bump Transactions](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0015.md) ([#278](https://github.com/stellar/java-stellar-sdk/pull/278)).
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,18 @@
...
+* Update SEP 10 implementation to reject fee-bump transactions and transactions with multiplexed addresses. (https://github.com/stellar/java-stellar-sdk/pull/278)
```
Comment (abuiles) [inline CHANGELOG.md:13]:
```suggestion
* Update SEP 10 implementation to reject fee-bump transactions and transactions with multiplexed addresses ([#278] (https://github.com/stellar/java-stellar-sdk/pull/278)).
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,18 @@
...
+* Update SEP 29 implementation to handle bump transactions. (https://github.com/stellar/java-stellar-sdk/pull/278)
```
Comment (abuiles) [inline CHANGELOG.md:14]:
```suggestion
* Update SEP 29 implementation to handle bump transactions ([#278](https://github.com/stellar/java-stellar-sdk/pull/278)).
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,18 @@
 
 As this project is pre 1.0, breaking changes may happen for minor version bumps. A breaking change will get clearly notified in this log.
 
+## 0.16.0
+
+* Update XDR definitions and auto-generated classes to support upcoming protocol 13 release. (https://github.com/stellar/java-stellar-sdk/pull/276)
```
Comment (ire-and-curses) [inline CHANGELOG.md:7]: One more of these

**Pull request merged**